src/EdFi.AnalyticsMiddleTier.Tests/Dimensions/When_querying_the_StudentProgramDim_view.cs

```diff
@@ -153,54 +167,95 @@ public void Then_should_have_StudentSchoolKey()
         }
 
         [SuppressMessage("ReSharper", "InconsistentNaming")]
-        public class Given_student_100023237
+        public class Given_student_190009_867530020_Special_Education_1684_778530
             : When_querying_the_StudentProgramDim_view
         {
-            private const string _caseIdentifier = "100023237";
-            public Given_student_100023237(TestHarness dataStandard) => SetDataStandard(dataStandard);
+            private const string _caseIdentifier = "190009_867530020_Special_Education_1684_778530";
+            public Given_student_190009_867530020_Special_Education_1684_778530(TestHarness dataStandard) => SetDataStandard(dataStandard);
 
             [Test]
-            public void Then_should_return_three_records()
+            public void Then_should_not_return_any_records()
             {
                 (bool success, string errorMessage) testResult =
                     DataStandard.RunTestCase<CountResult>(
-                        $"{TestCasesFolder}.{_caseIdentifier}_should_return_three_records.xml");
+                        $"{TestCasesFolder}.{_caseIdentifier}_should_not_return_any_records.xml");
                 testResult.success.ShouldBe(true, testResult.errorMessage);
             }
+        }
+
+        [SuppressMessage("ReSharper", "InconsistentNaming")]
+        public class Given_student_190257_867530023_Section_504_Placement_1637_867530
+            : When_querying_the_StudentProgramDim_view
+        {
+            private const string _caseIdentifier = "190257_867530023_Section_504_Placement_1637_867530";
+            public Given_student_190257_867530023_Section_504_Placement_1637_867530(TestHarness dataStandard) => SetDataStandard(dataStandard);
+
+            [Test]
+            public void Then_should_not_return_any_records()
+            {
+                (bool success, string errorMessage) testResult =
+                    DataStandard.RunTestCase<CountResult>(
+                        $"{TestCasesFolder}.{_caseIdentifier}_should_not_return_any_records.xml");
+                testResult.success.ShouldBe(true, testResult.errorMessage);
+            }
+        }
+
+        [SuppressMessage("ReSharper", "InconsistentNaming")]
+        public class Given_student_190257_867530023_Bilingual_1682_867530
+           : When_querying_the_StudentProgramDim_view
+        {
+            private const string _caseIdentifier = "190257_867530023_Bilingual_1682_867530";
+            public Given_student_190257_867530023_Bilingual_1682_867530(TestHarness dataStandard) => SetDataStandard(dataStandard);
 
             [Test]
-            public void Then_should_return_three_unique_programs()
+            public void Then_should_not_return_any_records()
             {
                 (bool success, string errorMessage) testResult =
                     DataStandard.RunTestCase<CountResult>(
-                        $"{TestCasesFolder}.{_caseIdentifier}_should_return_three_unique_programs.xml");
```

Comment:
If a Student can be in multiple programs, should we add back an additional test to test out them being in multiple programs like this previous test case we had?

Reply:
I added a new testcase to validate multiple programs.